tests/lambda_functions/vetext_incoming_forwarder_lambda/test_vetext_incoming_forwarder_lambda.py

```diff
@@ -35,71 +48,113 @@ def test_verify_parsing_of_twilio_message(event):
 
     assert response['statusCode'] == 200
     sqs_mock.assert_called_once()
 
-
 @pytest.mark.parametrize('event', [(albInvokedWithoutAddOn), (albInvokeWithAddOn), (sqsInvokedWithAddOn)])
-def test_failed_vetext_call_throws_general_exception(mocker, http_failure_response, event):
-    sqs_mock = mocker.patch(f'{lambda_module}.push_to_sqs')
-    mocker.patch(f'{lambda_module}.make_vetext_request', side_effect=Exception)
+def test_failed_vetext_call_throws_general_exception(mocker,  event):
+    sqs_mock = mocker.patch(f'{LAMBDA_MODULE}.push_to_sqs')
+    mocker.patch(f'{LAMBDA_MODULE}.read_from_ssm', return_value="ssm")
+    mocker.patch("http.client.HTTPSConnection")
+    mocker.patch("http.client.HTTPResponse", side_effect=Exception)
     response = vetext_incoming_forwarder_lambda_handler(event, None)
 
     assert response['statusCode'] == 200
     sqs_mock.assert_called_once()
 
-
 @pytest.mark.parametrize('event', [(sqsInvokedWithAddOn)])
 def test_failed_sqs_invocation_call_throws_general_exception(mocker, event):
-    sqs_mock = mocker.patch(f'{lambda_module}.push_to_sqs')
-    mocker.patch(f'{lambda_module}.process_body_from_sqs_invocation', side_effect=Exception)
+    sqs_mock = mocker.patch(f'{LAMBDA_MODULE}.push_to_sqs')
+    mocker.patch(f'{LAMBDA_MODULE}.process_body_from_sqs_invocation', side_effect=Exception)
     response = vetext_incoming_forwarder_lambda_handler(event, None)
 
     assert response['statusCode'] == 500
     sqs_mock.assert_not_called()
 
-
 @pytest.mark.parametrize('event', [(albInvokedWithoutAddOn), (albInvokeWithAddOn)])
 def test_failed_alb_invocation_call_throws_general_exception(mocker, event):
-    sqs_mock = mocker.patch(f'{lambda_module}.push_to_sqs')
-    mocker.patch(f'{lambda_module}.process_body_from_alb_invocation', side_effect=Exception)
+    sqs_mock = mocker.patch(f'{LAMBDA_MODULE}.push_to_sqs')
+    mocker.patch(f'{LAMBDA_MODULE}.process_body_from_alb_invocation', side_effect=Exception)
     response = vetext_incoming_forwarder_lambda_handler(event, None)
 
     assert response['statusCode'] == 500
     sqs_mock.assert_not_called()
 
-
 @pytest.mark.parametrize('event', [(albInvokedWithoutAddOn), (albInvokeWithAddOn)])
 def test_failed_alb_invocation_call_throws_key_error_exception(mocker, event):
-    sqs_mock = mocker.patch(f'{lambda_module}.push_to_sqs')
-    mocker.patch(f'{lambda_module}.process_body_from_alb_invocation', side_effect=KeyError)
+    sqs_mock = mocker.patch(f'{LAMBDA_MODULE}.push_to_sqs')
+    mocker.patch(f'{LAMBDA_MODULE}.process_body_from_alb_invocation', side_effect=KeyError)
     response = vetext_incoming_forwarder_lambda_handler(event, None)
 
     assert response['statusCode'] == 424
     sqs_mock.assert_not_called()
+
+@pytest.mark.parametrize('event', [(albInvokedWithoutAddOn), (albInvokeWithAddOn), (sqsInvokedWithAddOn)])
+def test_failed_ssm_paramter_retrieval_throws_general_exception(mocker,  event):
```

Comment:
The name is misleading.  You can test this by calling `read_from_ssm` directly and asserting that it returns the empty string.  You are also not really testing that an exception is raised because you catch it in the function, and the test doesn't see that.  If you didn't catch the exception, you could use [this](https://docs.pytest.org/en/stable/reference/reference.html?highlight=raises#pytest.raises).